apps/test/unit/p5lab/spritelab/SpritelabTest.js

```diff
@@ -145,6 +146,32 @@ describe('SpriteLab', () => {
         expect(resultingAnimations.orderedKeys.length).to.be.equal(1);
       });
 
+      describe('react to executionError', () => {
+        let alertSpy;
+        beforeEach(() => {
+          alertSpy = sinon.stub(studioApp(), 'displayWorkspaceAlert');
+          sinon.stub(instance, 'getMsg').returns({
+            workspaceAlertError: () => {
+              return 'translated string';
+            }
```

Comment:
we can simplify this expression with ES6 [arrow function expressions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)